appserver/pom.xml

```diff
@@ -90,7 +90,7 @@
 
         <!-- Jakarta Concurrency -->
         <jakarta.concurrent-api.version>3.1.1</jakarta.concurrent-api.version>
-        <concurro.version>3.1.0-M5</concurro.version>
+        <concurro.version>3.1.0-SNAPSHOT</concurro.version>
```

Comment:
Please revert to M5. The current master of Concurro has only one fix and some not very important changes since M5.